src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoNet.java

```diff
@@ -94,21 +94,17 @@ public NetworkComponent classifyLocation(@Nonnull Location l) {
             return null;
         }
 
-        switch (id) {
-            case "CARGO_MANAGER":
-                return NetworkComponent.REGULATOR;
-            case "CARGO_NODE":
-                return NetworkComponent.CONNECTOR;
-            case "CARGO_NODE_INPUT":
-            case "CARGO_NODE_OUTPUT":
-            case "CARGO_NODE_OUTPUT_ADVANCED":
-            case "CT_IMPORT_BUS":
-            case "CT_EXPORT_BUS":
-            case "CHEST_TERMINAL":
-                return NetworkComponent.TERMINUS;
-            default:
-                return null;
-        }
+        return switch (id) {
+            case "CARGO_MANAGER" -> NetworkComponent.REGULATOR;
+            case "CARGO_NODE" -> NetworkComponent.CONNECTOR;
+            case "CARGO_NODE_INPUT",
+                "CARGO_NODE_OUTPUT",
+                "CARGO_NODE_OUTPUT_ADVANCED",
+                "CT_IMPORT_BUS",
```

Comment:
ChestTerminal is not supported anymore 👀

Reply:
Want me to slip it in now or keep strictly on-scope?

Reply:
Can just do it now. Gonna be a breaking update for people who ignored warnings anyway.

Reply:
Done :)